src/extension/op_def.rs

```diff
@@ -445,13 +445,29 @@ impl Extension {
         name: SmolStr,
         description: String,
         signature_func: impl Into<SignatureFunc>,
+    ) -> Result<&mut OpDef, ExtensionBuildError> {
+        self.add_op_with_misc(name, description, signature_func, Default::default())
+    }
+
+    /// Add an operation definition to the extension. Must be a type scheme
+    /// (defined by a [`PolyFuncType`]), a type scheme along with binary
+    /// validation for type arguments ([`CustomValidator`]), or a custom binary
+    /// function for computing the signature given type arguments (`impl [CustomSignatureFunc]`).
+    ///
+    /// Includes a `misc` field for additional data that can be associated with the operation.
+    pub fn add_op_with_misc(
```

Comment:
this is what the `add_misc` method on `OpDef` is for, can you use that instead of adding a method?